app/src/main/java/org/mozilla/focus/settings/SettingsFragment.java

```diff
@@ -104,13 +106,30 @@ public void onAttach(Context context) {
     }
 
     @Override
-    public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
+    public void onCreateOptionsMenu(final Menu menu, MenuInflater inflater) {
```

Comment:
\[PMD\] INFO: The method 'onCreateOptionsMenu' has a Cyclomatic Complexity of 5\.